uaclient/entitlements/cc.py

```diff
@@ -32,14 +29,13 @@ def disable(self, silent=False, force=False):
         series = util.get_platform_info('series')
         repo_filename = self.repo_list_file_tmpl.format(
             name=self.name, series=series)
-        keyring_file = os.path.join(apt.APT_KEYS_DIR, self.repo_key_file)
         entitlement_cfg = self.cfg.read_cache(
             'machine-access-%s' % self.name)['entitlement']
         access_directives = entitlement_cfg.get('directives', {})
         repo_url = access_directives.get('aptURL', self.repo_url)
         if not repo_url:
             repo_url = self.repo_url
-        apt.remove_auth_apt_repo(repo_filename, repo_url, keyring_file)
+        apt.remove_auth_apt_repo(repo_filename, repo_url)
```

Comment:
Is this lacking a `fingerprint` arg now? It's defined as `remove_auth_apt_repo(repo_filename, repo_url, fingerprint)`

Reply:
Indeed, this needs fixing in all such cases, not just cc:
```
ubuntu@trusty-ua:~/git$ sudo ua disable esm
Traceback (most recent call last):
  File "/usr/bin/ua", line 9, in <module>
    load_entry_point('ubuntu-advantage-tools==18.1', 'console_scripts', 'ubuntu-advantage')()
  File "/usr/lib/python3/dist-packages/uaclient/cli.py", line 371, in main
    return args.action(args, cfg)
  File "/usr/lib/python3/dist-packages/uaclient/cli.py", line 162, in action_disable
    if entitlement.disable():
  File "/usr/lib/python3/dist-packages/uaclient/entitlements/esm.py", line 35, in disable
    apt.remove_auth_apt_repo(repo_filename, repo_url)
TypeError: remove_auth_apt_repo() missing 1 required positional argument: 'fingerprint'
ubuntu@trusty-ua:~/git$ 
```

Reply:
I also wonder why tests didn't fail.

We can also break this PR up into two: one to just handle the more pressing case of adding/removing the esm keyring, and the other about removing the superfluous keyrings.

Reply:
Tests passed because I correctly updated the mocked calls. When I rebased on master after the apt changes yesterday I accepted what was in master and did not update my calls in each of the entitlements.